src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpServer.cs

```diff
@@ -11,9 +11,10 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Hosting.Server;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Server.IISIntegration;
```

Comment:
Why do we have this reference? IAuthenticationSchemeProvider?

Reply:
`IISDefaults`